app/views/articles/index.html.erb

```diff
@@ -1,13 +1,46 @@
-<h1>Articles</h1>
-Our blog has <%= Article.public_count %>
-articles and counting!
-<ul>
-  <% @articles.each do |article| %>
-    <% unless article.archived? %>
-    <li>
-      <%= link_to article.title, article%>
-    </li>
-    <% end %>
-  <% end %>
-</ul>
-<%= link_to 'New Article', new_article_path %>
+
+<!-- Page Header -->
+<header class="masthead" style="background-image: url(<%= asset_path 'rails-bg.jpg' %>);">
+  <div class="overlay"></div>
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-8 col-md-10 mx-auto">
+        <div class="site-heading">
+          <h1>¡Articles on Rails!</h1>
+          <span class="subheading">Made with Ruby on Rails</span>
+        </div>
+      </div>
+    </div>
+  </div>
+</header>
+
+<!-- Main Content -->
+<div class="container">
+  <div class="row">
+    <div class="col-lg-8 col-md-10 mx-auto">
+      <% @articles.each do |article| %>
+        <% unless article.archived? or article.private? %>
+          <div class="post-preview">
+            <a href="<%= article_path article %>">
+                <h2 class="post-title">
+                  <%= article.title %>
+                </h2>
+            </a>
+            <p class="post-meta">Posted by
+              <a href="#">User on Rails</a>
+              on September 24, 2021
+            </p>
+          </div>
+        <% end %>
+      <% end %>
+      <hr>
+      <h3 class="post-subtitle" style="text-align:center;">
+      Our blog has <%= Article.public_count %> articles and counting!
+      </h3>
+    </div>
+  </div>
+</div>
+
+
+
+
```

Comment:
Please delete the extra lines at the end of the file.

Reply:
Done!